jobsdb/migration.go

```diff
@@ -223,29 +274,53 @@ func (jd *HandleT) cleanupStatusTables(ctx context.Context, dsList []dataSetT) e
 				return err
 			}
 		}
+		for _, statusTable := range toVaccum {
+			if err := jd.vaccumStatusTable(
+				ctx,
+				tx,
+				statusTable.JobStatusTable,
+			); err != nil {
+				return err
+			}
+		}
 		return nil
 	})
 }
 
 // cleanStatusTable deletes all rows except for the latest status for each job
 func (*HandleT) cleanStatusTable(ctx context.Context, tx *Tx, table string) error {
-	_, err := tx.ExecContext(
+	deletedJobsQuery := fmt.Sprintf(`WITH deleted AS (DELETE FROM %[1]q
```

Comment:
I don't think this is necessary, as a delete statement returns the number of rows affected already